### ISSUE TITLE
Ifs do not inherit scoping

### ISSUE THREAD

**Issue by vihanb** [cheddar-lang/Cheddar#2]
**Version**: 0.0.316-alpha
**System**: Ubuntu

Given a simple variable:

```js
var a := 5
```

if statements fail to work when a variable is referenced:

```js
if a == 4 {
    print "a is 4"
} else {
    print "a is not 4"
}
```

In which "a is 4" is printed where it is actually not.

**Comment by vihanb** [comment]
Appears to be caused by the misnaming of the passed scope. The `if`s `constructor` looks like this:

```js
constructor(tokens, scope) {
    this.tokens = tokens;
    this.scope - scope;
}
```

But the scope is referenced as `this.Scope`. Causing rather confusing bugs.

---

Resolution is to:

A) Temporarily fix case
B) Establish standard on which to name variables

**Comment by vihanb** [comment]
Fox has been made. Will be available in hotfix v0.0.317-alpha

**Comment by RikerW** [comment]
Yes, I have to point out the typo. `s/Fox/fix/`
